### PR TITLE
update arm and hand examples

### PR DESCRIPTION
**Pull request by georgiablanco**
## Proposed changes

update arm and hand examples

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [x] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [x] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

**Review by biofotis — APPROVED** [review]

File: sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py [hunk 1]
```diff
@@ -55,24 +60,54 @@
...
+                         'rh_THJ5': 0.17, 'rh_FFJ1': 1.5707, 'rh_FFJ2': 1.5707, 'rh_FFJ3': 1.5707,
+                         'rh_FFJ4': 0.0, 'rh_MFJ1': 1.5707, 'rh_MFJ2': 1.5707, 'rh_MFJ3': 1.5707,
+                         'rh_MFJ4': 0.0, 'rh_RFJ1': 1.5707, 'rh_RFJ2': 1.5707, 'rh_RFJ3': 1.5707,
+                         'rh_RFJ4': 0.0, 'rh_LFJ1': 1.5707, 'rh_LFJ2': 1.5707, 'rh_LFJ3': 1.5707,
+                         'rh_LFJ4': 0.0, 'rh_LFJ5': 0.0, 'rh_WRJ1': 0.0, 'rh_WRJ2': 0.0}
 
 arm_hand_home_joints_goal = {'ra_shoulder_pan_joint': 0.00, 'ra_elbow_joint': 2.00,
                              'ra_shoulder_lift_joint': -1.25, 'ra_wrist_1_joint': -0.733,
-                             'ra_wrist_2_joint': 1.5708, 'ra_wrist_3_joint': 0.00,
+                             'ra_wrist_2_joint': 1.5708, 'ra_wrist_3_joint': -3.1416,
```
Comment (bdenoun) [inline sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py:74]: Why this change? I mean, -3.1416 is the same as 0.0001 right?

Comment (georgiablanco) [reply sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py:74]: When we updated the UR repository, the eef was rotated by pi. This wasn't changed to reflect this hence why I am making the change now.

Comment (bdenoun) [reply sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py:74]: Alright thanks :)

**Review by bdenoun — APPROVED** [review]
